examples/chat/enterprise/aws-bedrock-anthropic/app_utils.py

```diff
@@ -0,0 +1,27 @@
+import os
+from pathlib import Path
+
+app_dir = Path(__file__).parent
+env_file = app_dir / ".env"
+
+
+def load_dotenv(dotenv_path: os.PathLike[str] = env_file, **kwargs) -> None:
+    """
+    A convenience wrapper around `dotenv.load_dotenv` that warns if `dotenv` is not installed.
+    It also returns `None` to make it easier to ignore the return value.
+    """
+    try:
+        import dotenv
+
+        _ = dotenv.load_dotenv(dotenv_path=dotenv_path, **kwargs)
+        return None
```

Comment:
Small nit: I suggest removing both `return` statements, or at least changing this to just `return` for consistency with the one below.